R/helpers.R

```diff
@@ -214,3 +214,22 @@ var_encode <- function(data,
     })
   )
 }
+
+# Yardstick doesn't currently include MdAPE, so we'll add it here
+mdape_vec <- function(truth, estimate, case_weights = NULL, na_rm = TRUE) {
+  check_numeric_metric(truth, estimate, case_weights)
```

Comment:
**nitpick:** Let's namespace the functions used inside of helper functions, i.e.

```suggestion
  yardstick::check_numeric_metric(truth, estimate, case_weights)
```